pkg/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Can we remove this change?
It's an autogenerated file so next time we generate it will show up as a new change